### PR TITLE
PathOpGui.py Turning Addon edit.

### PR DESCRIPTION
**Pull request by WarrenKCash**
Thank you for creating a pull request to contribute to FreeCAD! To ease integration, we ask you to conform to the following items. Pull requests which don't satisfy all the items below might be rejected. If you are in doubt with any of the items below, don't hesitate to ask for help in the [FreeCAD forum](https://forum.freecadweb.org/viewforum.php?f=10)!

- [x]  Your pull request is confined strictly to a single module. That is, all the files changed by your pull request are either in `App`, `Base`, `Gui` or one of the `Mod` subfolders. If you need to make changes in several locations, make several pull requests and wait for the first one to be merged before submitting the next ones
- [ ]  In case your pull request does more than just fixing small bugs, make sure you discussed your ideas with other developers on the FreeCAD forum
- [ ]  Your branch is [rebased](https://git-scm.com/docs/git-rebase) on latest master `git pull --rebase upstream master`
- [ ]  All FreeCAD unit tests are confirmed to pass by running `./bin/FreeCAD --run-test 0`
- [ ]  All commit messages are [well-written](https://chris.beams.io/posts/git-commit/) ex: `Fixes typo in Draft Move command text`
- [ ]  Your pull request is well written and has a good description, and its title starts with the module name, ex: `Draft: Fixed typos`
- [ ]  Commit messages include `issue #<id>` or `fixes #<id>` where `<id>` is the issue ID number from our [Issues database](https://github.com/FreeCAD/FreeCAD/issues) in case a particular commit solves or is related to an existing issue. Ex: `Draft: fix typos - fixes #4805`

And please remember to update the Wiki with the features added or changed once this PR is merged.  
**Note**: If you don't have wiki access, then please mention your contribution on the [0.20 Changelog Forum Thread](https://forum.freecadweb.org/viewtopic.php?f=10&t=56135).

---

### CODE REVIEW

**Comment by WarrenKCash** [comment]
The link to related FreeCAD forum post [https://forum.freecadweb.org/viewtopic.php?style=5&f=15&t=30563&start=150](url)

**Comment by luzpaz** [comment]
@WarrenKCash thanks for braving `git`

**Comment by berndhahnebach** [comment]
<a href="https://gitlab.com/freecad/FreeCAD-CI/-/commits/PR_7269">pipeline status</a> for feature branch [PR_7269](https://gitlab.com/freecad/FreeCAD-CI/-/commits/PR_7269). Pipeline [600469887](https://gitlab.com/freecad/FreeCAD-CI/-/pipelines/600469887) was triggered at [e4ea88a](https://github.com/FreeCAD/FreeCAD/pull/7269/commits/e4ea88ada8745ac6a1eaa3e2a87ea610b1443ae0). All CI [branches](https://gitlab.com/freecad/FreeCAD-CI/-/branches/all) and [pipelines](https://gitlab.com/freecad/FreeCAD-CI/-/pipelines?scope=branches).

File: src/Mod/Path/PathScripts/PathOpGui.py [hunk 1]
```diff
@@ -1375,7 +1375,7 @@ def Create(res):
     that is created in each operations Gui implementation."""
     FreeCAD.ActiveDocument.openTransaction("Create %s" % res.name)
     try:
-        obj = res.objFactory(res.name, obj=None, parentJob=res.job)
+        obj = res.objFactory(res.name, obj=None)#, parentJob=res.job)
```
Comment (mlampert) [inline src/Mod/Path/PathScripts/PathOpGui.py:1378]: Sorry for being late to the party. This is not the best way of avoiding the issue and this change breaks creation of ops when there are multiple jobs in a single file.

The much better, and correct way of dealing with the issue is add the `parentJob` named parameter to all the `Create(..)` functions of the Ops in _FreeCAD_Turning_Addon_ .

FYI: the additional parameters for `Create(..)` came in ~ a year ago with commit #4797

**Review by mlampert — CHANGES_REQUESTED** [review]

File: src/Mod/Path/PathScripts/PathOpGui.py (hunk 1 above)
Comment (luzpaz) [reply src/Mod/Path/PathScripts/PathOpGui.py:1378]: @WarrenKCash ping

**Comment by sliptonic** [comment]
Per @mlampert's comment, this should be fixed in liblathe.   I will close this PR in a few days unless I hear back.

**Comment by WarrenKCash** [comment]
Hello everyone thank for the help. As @mlampert's suggested I have added the parent job to the end of each turning op's in lib lathe and it seems to function as intended  with the original pathOpGui.py file in FreeCAD. I will post the files on the FreeCAD Path LibLathe forum discussion for others to test.
  @sliptonic please close this request.

**Comment by sliptonic** [comment]
closing per discussion

**Pull request closed without merging**